classes/search/block/sql/class-sqlquery.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Block search SQL query
+ *
+ * @package P4BKS\Search\Block
+ */
+
+namespace P4GBKS\Search\Block\Sql;
+
+use P4GBKS\Search\Block\Query;
+use P4GBKS\Search\Block\Query\Parameters;
```

Comment:
Maybe now there's 2 kinds of parameters, we can make the distinction clearer by either renaming this one, or importing only the `P4GBKS\Search\Block` namespace, so that inside this file it looks like `new Block\Query\Parameters`.